packages/loot-core/src/server/accounts/rules.js

```diff
@@ -456,7 +457,7 @@ export class Rule {
       return false;
     }
 
-    return this.conditions.every(condition => {
+    return this.conditions[this.quantifier](condition => {
```

Comment:
I’d rather this be explicit so that invalid values produce an error rather than calling arbitrary array methods.

Reply:
I thought model validation would prevent values that are not whitelisted, but happy to add an explicit check